cli/azd/internal/vsrpc/models.go

```diff
@@ -68,6 +68,12 @@ func newInfoProgressMessage(message string) ProgressMessage {
 	}
 }
 
+func (m ProgressMessage) Fill(message string) ProgressMessage {
```

Comment:
FWIW - Took me a moment to realize that `Fill` would not actually mutate this instance and instead returns a new instance with the given message and Time.Now().  While the signature makes it clear, to me `Fill` evokes the idea that we would mutate this instance.  I'm not if `CopyWithMessage` is a "better" name, maybe just a helpful doc comment about how this returns a new instance?

Reply:
Updated to `WithMessage`